slither/utils/expression_manipulations.py

```diff
@@ -91,43 +106,85 @@ def copy_expression(
         # (.. ? .. : ..).add
         if isinstance(expression, MemberAccess):
             next_expr = expression.expression
-            if self.apply_copy(next_expr, true_expression, false_expression, f_expression):
+            if self.conditional_not_ahead(
+                next_expr, true_expression, false_expression, f_expression
+            ):
                 self.copy_expression(
                     next_expr, true_expression.expression, false_expression.expression
                 )
-
+        # pylint: disable=too-many-nested-blocks
         elif isinstance(expression, (AssignmentOperation, BinaryOperation, TupleExpression)):
             true_expression._expressions = []
             false_expression._expressions = []
-
             for next_expr in expression.expressions:
-                if isinstance(next_expr, IndexAccess):
-                    # create an index access for each branch
-                    if isinstance(next_expr.expression_right, ConditionalExpression):
-                        next_expr = _handle_ternary_access(
-                            next_expr, true_expression, false_expression
+                # TODO: can we get rid of `NoneType` expressions in `TupleExpression`?
```

Comment:
Not sure, my intuition tells me that could happen with unnamed tuple (ex: `(,,,) = f()`). Through I haven't tested, so I could be wrong